index.html

```diff
@@ -14,7 +14,7 @@
 
 	<nav class="navbar navbar-default">
 		<div class="container-fluid">
-			<div class="imageItem"><a><img class="helpImage" src="images/Help.png"></a></div>
+			<div class="image-item"><a><i class="glyphicon glyphicon-question-sign"></i></a></div>
```

Comment:
<i> is an old variant for bootstrap 2. Since we are using v.3 all icons should be included as a <span>